### PR TITLE
replaced deprecated context.getFlashCookie() with context.getFlashScope

### PR DESCRIPTION
**Pull request by joshkamau**
replaced deprecated context.getFlashCookie() with context.getFlashScope() in ninja-servlet-jpa-blog-archetype

### CODE REVIEW

File: nb-configuration.xml [hunk 1]
```diff
@@ -1,24 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project-shared-configuration>
-    <!--
```
Comment (raphaelbauer) [inline nb-configuration.xml:3]: Was there a specific reason to delete this file?

File: ninja-servlet-jpa-blog-archetype/src/main/resources/archetype-resources/src/main/java/controllers/ArticleController.java [hunk 2]
```diff
@@ -76,10 +76,10 @@ public Result articleNewPost(@LoggedInUser String username,
                                  Validation validation) {
 
         if (validation.hasViolations()) {
-
-            context.getFlashCookie().error("Please correct field.");
-            context.getFlashCookie().put("title", articleDto.title);
-            context.getFlashCookie().put("content", articleDto.content);
+            
+            context.getFlashScope().error("Please correct field.");
```
Comment (raphaelbauer) [inline ninja-servlet-jpa-blog-archetype/src/main/resources/archetype-resources/src/main/java/controllers/ArticleController.java:80]: Much better :+1:

**Comment by raphaelbauer** [comment]
Hi Josh, 

thanks for the PR :) Very useful to keep everything up to date and refactor deprecated method calls. Would you mind checking the issue with nb-configuration.xml?

Cheers! 

Raphael

**Comment by jlannoy** [comment]
Replaced by PR #633

**Pull request closed without merging**